doc/src/quickstart/index.rst

```diff
@@ -308,17 +280,18 @@ The corresponding command to run is:
 .. code-block:: bash
 
     > $VTR_ROOT/abc/abc \
-        -c 'read blink.odin.blif; if -K 6; write_hie blink.odin.blif blink.abc_no_clock.blif'
+        -c 'read ~/vtr_work/quickstart/blink_manual/temp/blink.parmys.blif; if -K 6; write_hie ~/vtr_work/quickstart/blink_manual/temp/blink.parmys.blif ~/vtr_work/quickstart/blink_manual/temp/blink.abc_no_clock.blif'
+
 
 When run, ABC's output should look similar to::
 
-    ABC command line: "read blink.odin.blif; if -K 6; write_hie blink.odin.blif blink.abc_no_clock.blif".
+    ABC command line: "read blink.parmys.blif; if -K 6; write_hie blink.parmys.blif blink.abc_no_clock.blif".
 
     Hierarchy reader converted 6 instances of blackboxes.
     The network was strashed and balanced before FPGA mapping.
     Hierarchy writer reintroduced 6 instances of blackboxes.
 
-If we now inspect the produced BLIF file (``blink.abc_no_clock.blif``) we see that ABC was able to significantly simplify and optimize the circuit's logic (compared to ``blink.odin.blif``):
+If we now inspect the produced BLIF file (``blink.abc_no_clock.blif``) we see that ABC was able to significantly simplify and optimize the circuit's logic (compared to ``blink.parmys.blif``):
```

Comment:
Take a look at the two files and see if this is true.  Are the number of .names significantly different?  Confirm they are different a bit at least.  We shouldn't say it is significantly simpler / optimized unless it is.

Reply:
Thanks.  I think we should just simplify then to show running parmys in this section, without abc or the clock relabelling (we can leave those in the Odin-II section). If you can confirm the clock relabeling is *not* called for the parmys flow in run_vtr_flow that would be ideal -- then we can confidently remove it from this section.

Reply:
sure thing. It does not appear to be called in run_vtr_flow. 

My initial confusion stemmed from reading the Parmys paper, which states, "the Parmys pass is called to perform partial mapping according to the target architecture, and then a series of optimization and logic mappings are performed to convert all the remaining Yosys internal types to primitives. Next, the output is passed to ABC and VPR stages."

This led me to believe that running only the Parmys part of the flow would require a separate invocation of ABC since we are executing the command with both -start parmys and -end parmys, it explicitly skips the ABC.

I will update the PR and also wait for final confirmation by the added members

Reply:
I think you were right and I was wrong @ZohairZaidi !  After talking to Ken Kent at UNB, it does seem we need to run ABC and the clock add/removal script after Parmys.  So I think your original edits were right.  Can you run the flow to check exactly what happens in run_vtr_flow with parmys?   I think it runs ABC and the clock recovery script after all, so we should document them again.

I also think we should change the order of the text.  After saying how to run vpr, I think we should say how to use run_vtr_flow with parmys to synthesize, place and route a circuit. Then we can have a "running manually" section to dig more into how to run each tool.  Finally we can have the existing Odin II section.